winlogbeat/Jenkinsfile.yml

```diff
@@ -29,6 +29,11 @@ stages:
         platforms:             ## override default labels in this specific stage.
             - "windows-2019"
         stage: mandatory
+    windows-2022:
```

Comment:
Are we waiting for a fix to https://github.com/elastic/beats/issues/30621 before enabling this for winlogbeat, or should this be removed from the initial PR until the failure is fixed?

Reply:
Aha! I somehow missed this one, so let me move this section away :)

Reply:
Since the `x-pack/winlogbeat` work completely fine